src/main/resources/static/js/survey.js

```diff
@@ -65,12 +65,16 @@ function makeQuestionDiv() {
     // When a new question input div is create the question type
     // is stored as the class of the div
     switch (type) {
-    case "openEnded":
-        qDiv.append($("<input type='text'/>"));
-        qDiv.addClass('openEnded');
-        break;
-    default:
-        break;
+        case "numberQuestion":
```

Comment:
The input types would ideally be numbers, though I guess you wanted to have the words min and max there to clarify the purpose.
If you want to stick with text inputs, there should be a validation stage to check if they are numbers before sending to the server.